core/quantity/src/test/CompositeFormats.test.ts

```diff
@@ -83,11 +83,11 @@ describe("Composite Formats tests:", () => {
       assert.isTrue(format.hasUnits);
 
       const testEntry = {
-        // SWB
+        // SWB What does context mean here?
```

Comment:
contextId here has nothing to do with iModel .... it basically mean "unit family" like Length, Area, Pressure, Temperature etc.

Reply:
I think we should leave it be for now.... we just need to remove all those // SWB .... comments.

Reply:
Yes, the two outstanding renaming PRs will be merged together to check consistency then all `// SWB` tags will be removed